util/test/sub_test.py

```diff
@@ -1570,7 +1570,7 @@ def filter_errors(output_in, pre_exec_output, execgoodfile, execlog):
         # Run the precompile script
         #
         if globalPrecomp:
-            sys.stdout.write('[Executing ./PRECOMP]\n')
+            sys.stdout.write('[Executing ./PRECOMP %s %s %s]\n'%(execname, complog. compiler))
```

Comment:
Shouldn't that be a comma after complog?

Reply:
Your eyes are better than mine (strangely, though, at one point I had it in a form that it worked because I copy-pasted the line to run it myself...  I must have changed it in two workspaces and checked in the wrong one?)  Thanks!